_data/releases.yaml

```diff
@@ -1,3 +1,50 @@
+- version: 22.0.0.Final
+  version_shortname: 22.0.0
```

Comment:
@asoldano How about "22 Final"

This string drives what is displayed in the big font text on the wildlfy.org landing page.  22.0.0 is ok though.

Reply:
I'll tweak that in the commit I do to add WildFly Preview.